src/pip/_internal/utils/glibc.py

```diff
@@ -50,7 +63,7 @@ def glibc_version_string_ctypes() -> Optional[str]:
 
     # Call gnu_get_libc_version, which returns a string like "2.5"
     gnu_get_libc_version.restype = ctypes.c_char_p
-    version_str = gnu_get_libc_version()
+    version_str: str = gnu_get_libc_version()
```

Comment:
I just copied over the full `_glibc_version_string_ctypes` from `_manylinux.py`, and this was the only other change. I can back it out if desired.